.vault-config/productconstructiondev.yaml

```diff
@@ -0,0 +1,38 @@
+storageLocation:
```

Comment:
Can we name these `product-construction-dev.yaml` so it's more readable? Same for the other file.

Reply:
sure, I just kind of kept using the same pattern that we had